deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/JobsParser.java

```diff
@@ -17,6 +17,7 @@
 /**
  * Representation of a single Jobs collection with a Json serializer.
  */
+@SuppressWarnings("FieldCanBeLocal")
```

Comment:
Suppressing at class level for all fields used for serialization below.

Reply:
how about adding this statement as a code comment?

Reply:
+1 to this point. These suppressions should have explanations in code when it isn't obvious why we are suppressing them

Reply:
sure will do

Reply:
How about "This field cannot be local because json serialization requires it to be declared at the class level" instead?